build.gradle

```diff
@@ -35,6 +35,8 @@ dependencies {
 	testCompileOnly 'org.projectlombok:lombok:1.18.16'
 	testAnnotationProcessor 'org.projectlombok:lombok:1.18.16'
 	compileOnly('org.modelmapper:modelmapper:1.1.0')
+	compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.7.0'
```

Comment:
Try to be consistent with the dependency declaration, see how other ones are declared